Containers: Minor improvements to EKS guide (#31)
CrowdStrike/Cloud-AWS: Container Security/implementation-guide.md

   injector-6499dbd4b5-v5gqr   1/1     Running   0          2d3h
   ```

>  - (optional) Run the installer without any command-line arguments to get sense of configuration options are available for the deployment.

Reviewer: ```suggestion
 - (optional) Run the installer without any command-line arguments to output available configuration options for the deployment.
```

   ```

>    $ docker run --rm --entrypoint installer $FALCON_IMAGE_URI

Reviewer: ```suggestion
   $ docker run --rm --entrypoint installer --cid $FALCON_CID -image $FALCON_IMAGE_URI
```

Reviewer: It appears ``-cid`` is mandatory. Should the example CLI be updated?

Author: You are right, `--cid` is required. But as soon as You give it cid, it won't shown the options. I will reword this a bit to mention `--help` instead of adding`--cid` here.

Reviewer: If two dashes are needed (``--cid`` vs ``-cid``) then the help output should be updated as well

Author: Oh, both `-cid` and `--cid` works. It is super awesome. Anyway, this demo is rock solid, no need to update anything. 😉
